.github/workflows/publish.yml

```
@@ -16,26 +16,28 @@
 
-      - name: Build and push Docker image
+      - name: "Build and push Docker image"
         uses: docker/build-push-action@v5
         with:
-          context: .
+          context: ./apps/cricketscoreapi
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)